saleor/plugins/webhook/plugin.py

```diff
@@ -211,12 +211,20 @@ def checkout_updated(self, checkout: "Checkout", previous_value: Any) -> Any:
             WebhookEventType.CHECKOUT_UPDATED, checkout_data
         )
 
-    def notify(self, event: "NotifyEventType", payload: dict, previous_value) -> Any:
+    def notify(self, event: NotifyEventType, payload: dict, previous_value) -> Any:
         if not self.active:
             return previous_value
+
+        notify_user_event = WebhookEventType.NOTIFY_USER
         data = {"notify_event": event, "payload": payload}
+
+        if event not in NotifyEventType.CHOICES:
+            logger.warning(
```

Comment:
Why warning? This could produce a massive logs

Reply:
What about this comment? How should we address it @korycins @mstrumeck?

Reply:
Sorry, I thought that I've changed that into `logger.info`